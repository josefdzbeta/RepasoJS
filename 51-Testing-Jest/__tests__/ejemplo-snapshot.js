const cliente = {
    nombre: 'Jose Angell',
    sueldo: 500,
    tipo: 'Premium'
}

describe('testing al cliente', ()=>{
    test('Es Jose Angel',()=>{
        expect(cliente).toMatchSnapshot();
    });
});