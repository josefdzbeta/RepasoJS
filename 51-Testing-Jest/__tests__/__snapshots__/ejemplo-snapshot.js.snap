// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing al cliente Es Jose Angel 1`] = `
{
  "nombre": "Jose Angell",
  "sueldo": 500,
  "tipo": "Premium",
}
`;
